arctic/scripts/arctic_delete_library.py

```diff
@@ -28,7 +28,7 @@ def main():
     if not opts.library:
         parser.error('Must specify the full path of the library e.g. arctic_jblackburn.lib!')
 
-    print "Deleting: %s on mongo %s" % (opts.library, opts.host)
+    print("Deleting: %s on mongo %s" % (opts.library, opts.host))
```

Comment:
I could have done that, but python2.7 supports print as a function, i.e. print()